### ISSUE TITLE
Variant annotation fails to annotate without warning

### ISSUE THREAD

**Issue by hassanfa** [Ensembl/ensembl-vep#698]
## Describe the issue
VEP fails to annotate a VCF of 19 variants and output file ends up with zero variants. This VCF is a mixture of INDELs and structural variants.

The problem is when individual variants are given to VEP, some get successfully annotated.

I have tried:
--allow_non_variant
--dont_skip

I have also tried this on all major releases of VEP from version 94 to 99, and this happens on all of them.

Attached file is produced from NA12878 dataset using VarDict to produce this error and it is safe to share.
[SNV.somatic.TL001_NL002.vardict.other.chr13.vcf.gz](https://github.com/Ensembl/ensembl-vep/files/4245560/SNV.somatic.TL001_NL002.vardict.other.chr13.vcf.gz)

## Additional information
Please fill in the following sections to help us find the source of your issue as quickly as possible.

### System
- VEP version: 99
- VEP Cache version: 99
- Perl version: 5.26.2
- OS: CentOS
- tabix installed: yes

Addition:
```
Versions:
  ensembl              : 99.d3e7d31
  ensembl-funcgen      : 99.0832337
  ensembl-io           : 99.441b05b
  ensembl-variation    : 99.642e1cd
  ensembl-vep          : 99.1
```

cache info:
```
HGMD-PUBLIC	20174
sift	sift5.2.2
ClinVar	201810
gnomAD	r2.1
refseq	01_2015
assembly	GRCh37.p13
genebuild	2011-04
regbuild	1.0
dbSNP	151
1000genomes	phase3
COSMIC	86
gencode	GENCODE 19
polyphen	2.2.2
ESP	20141103
```

### Full VEP command line
```
vep \
--input_file SNV.somatic.TL001_NL002.vardict.other.chr13.vcf.gz \
--dir_cache ~/reference/GRCh37/vep/ --format vcf --force_overwrite  \
--format vcf --force_overwrite  \
--verbose --cache --offline --merged --everything \
--no_stats --assembly GRCh37 --vcf --output_file STDOUT
```

When only a variant or two is given, then annotation works just fine:
```
bcftools view -r 13:25671809 SNV.somatic.TL001_NL002.vardict.other.chr13.vcf.gz | \
vep \
--dir_cache ~/reference/GRCh37/vep/ --format vcf --force_overwrite  \
--format vcf --force_overwrite  \
--verbose --cache --offline --merged --everything \
--no_stats --assembly GRCh37 --vcf --output_file STDOUT
```

### Full error message
> Including the warnings, if available

### Data files (if applicable)
They include:
- The input file: attached is the smallest dataset that I could create to replicate above.
~- The output file~
~- The custom file(s)~
~You can provide a sample of them~

**Comment by hassanfa** [comment]
This might be related to #677 , the difference is my input file VEP can successfully annotate some but it doesn't unless it is explicitly given those variants. See example 2 in "Full VEP command line" in issue text body.

**Comment by dglemos** [comment]
Hi @hassanfa,

Thanks for the detailed explanation, it's always very helpful to understand the issue. 
Could you try to run only the non-structural variants from your input file? VEP should annotate these variants without any problem. 
Your problem seems to be with the SV deletions. The INFO column doesn't have the end coordinate (END) or even the length of the variant (SVLEN) necessary to annotate the variants. You can read the [VCF format documentation](http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/VCF%20%28Variant%20Call%20Format%29%20version%204.0/encoding-structural-variants) on how to describe structural variants in VCF format.

**Comment by hassanfa** [comment]
Thanks for the quick reply Diana.

I looked into the following:
1. After removing all SVTYPE <DEL> , VCF file was annotated successfully.
2. I created another VCF file which has DEL, INV, DUP, and some INDELs. Only DEL variants and all the variants after DEL were not annotated. Again, when i removed SVTYPE=DEL, all of them were successfully annotated.

Input VCF is clearly badly formatted from source for SVTYPE={DEL,INV,DUP}, that is clear. The part I don't fully grasp yet is why does VEP skip the rest of VCF when it encounters a variant with a missing INFO-tag? Is it possible to override this or simply not annotate that particular variant?
